src/openzaak/fixtures/default_admin_index.json

```diff
@@ -36,6 +36,10 @@
                 "authorizations",
                 "applicatie"
             ],
+            [
+                "autorisaties",
+                "catalogusautorisatie"
```

Comment:
On the second thought should we hide them from the dashboard? We don't expect users to manually create and delete them, right?